SYCL/Plugin/interop-opencl-interop-task-mem.cpp

```diff
@@ -0,0 +1,52 @@
+// REQUIRES: opencl
```

Comment:
```suggestion
// REQUIRES: opencl, opencl_icd
```
OpenCL ICD loader is also needed to resolve direct OpenCL calls